Automattic/jetpack
# VideoPress: handle how the VideoCard looks when no data is provided

This PR introduces the `isBlank` property to the `<VideoCard />` component. When True, the component hides its whole content but doesn't lose the dimms. 
Also, when thumbnail property is not defined, the component shows a Video Icon.

Fixes #

#### Changes proposed in this Pull Request:

*

#### Other information:

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to StoryBook
* Use the tools to check the 

https://user-images.githubusercontent.com/77539/189202185-ed7df3b0-dfca-4ccf-8193-5a0283eaaa84.mov

size and show/hide the placeholder when there is no thumbnail
